docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -475,6 +476,7 @@ Note: This feature is supported only in NGINX Plus.
 |``domain`` | The domain for which the cookie is set. | ``string`` | No |
 |``httpOnly`` | Adds the ``HttpOnly`` attribute to the cookie. | ``boolean`` | No |
 |``secure`` | Adds the ``Secure`` attribute to the cookie. | ``boolean`` | No |
+|``samesite`` | Adds the ``SameSite`` attribute to the cookie. The allowed values are: ``strict``, ``lax``, ``none`` | ``string`` | No |
```

Comment:
I have a question about the cases. If the values are case-sensitive in NGINX, and we document as such here, why do we need to accept uppercase and capitalised values, and then convert it to lowercase both in validation and in the template? Seems like unnecessary effort for runtime and testing?